pkgs/development/python-modules/bqscales/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, hatchling
+, jupyter-packaging
+, pytestCheckHook
+, ipywidgets
+, numpy
+, traitlets
+, traittypes
+}:
+
+buildPythonPackage rec {
+  pname = "bqscales";
+  version = "0.3.1";
+
+  format = "pyproject";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-C+/GLpqYpePngbn5W0MwvpdmVgFZF7aGHyKMgO5XM90=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+    jupyter-packaging
+  ];
+
+  propagatedBuildInputs = [
+    ipywidgets
+    numpy
+    traitlets
+    traittypes
+  ];
+
+  doCheck = false;
+  # no tests in PyPI dist
```

Can we use `fetchFromGitHub`?
```suggestion
  # no tests in PyPI dist
  doCheck = false;
```

---

Switched.